docs/docs/quickstart.mdx

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 3
+title: Quickstart
+---
+
+# Quickstart
+
+Follow these steps for a quick start:
```

Comment:
It would be great to have at the top a summary of what the user is going to perform